scripts/goreleaser.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env sh
+# Wrap goreleaser by using envsubst on .goreleaser.yml,
```

Comment:
can you copy the latest from insights-plugins? I made some formatting changes.

Also--do we need this script here? Polaris tagging strategy should be much more normal than the monorepo

Reply:
@rbren We do need this script to push docker manifests for feature branches. How does the indenting look now - this script differs from that of insights-plugins, so I fixed indenting by hand.